tests/test_routes.py

```diff
@@ -129,6 +129,21 @@ def test_create_customer(self):
         # self.assertEqual(new_customer["email"], test_customer.email)
         # self.assertEqual(new_customer["password"], test_customer.password)
 
+    def test_index(self):
+        """ It should call the home page """
+        resp = self.app.get("/")
+        self.assertEqual(resp.status_code, status.HTTP_200_OK)
+        
+    def test_delete_customer(self):
+        """It should Delete a Customer"""
+        test_customer = self._create_customers(1)[0]
+        response = self.client.delete(f"{BASE_URL}/{test_customer.id}")
+        self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
+        self.assertEqual(len(response.data), 0)
+        # make sure they are deleted
+        response = self.client.get(f"{BASE_URL}/{test_customer.id}")
```

Comment:
This is failing because the POST request does not currently return the newly created Customer record (once we finish the GET endpoint this will not be the case).

To work around this, we can query the database with the Customer model using the `Customer.find_by_email` class method, since the `email` attribute is unique. This method returns a list of customers. So essentially the test steps after you created the test_customer in ln 105 would be:

1. use the `Customer` class to find instances of customers with the same email as `test_customer`
2. make sure that the returned query list is not empty and only returns 1 record
3. get that record that was returned from the DB
4. use the id of that customer to call the delete method and make sure the status code is as expected
5. NOW, use `find_by_email` to search for the record in the DB again
6. make sure no records get returned